### ISSUE TITLE
Не отображать в параметрах построения отключенные записи

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2768]
Originally reported on Google Code with ID 2768

```

    Добавьте в параметры отчетов галку "Не отображать в параметрах построения отключенные
записи".
    При задании параметров отчета приходиться по памяти подбирать некоторые значения
(среди всех значений ТОЛЬКО актуальные значения, не отключенные). Например, подразделения,
статьи затрат и т. д.

```

Reported by `NikolayUkleyko` on 2012-03-30 15:08:35

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-03-30 15:37:27
- **Labels added**: Type-Enhancement, Priority-Medium, GedeminEXE, Report
- **Labels removed**: Broiler, Gomeloboi, Etalon

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-07-27 11:21:56

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-05-06 12:40:55
- **Status changed**: `Duplicate`

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-05-06 12:41:39
- **Merged into**: #2676
